dts/bindings/stepper/step-dir-timing.yaml

```diff
@@ -0,0 +1,12 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025 Jilay Sandeep Pandya
+# SPDX-License-Identifier: Apache-2.0
+
+description: Step/Dir Control Signal Timing
+
+properties:
+  step-width-ns:
+    type: int
+    required: true
+    description: |
+      Total pulse width in nanoseconds for the step signal.
+      This is the total duration for which the step signal is asserted high and low.
```

Comment:
Not sure if the description is correct, this is what the timer effectively does:
<img width="810" height="237" alt="image" src="https://github.com/user-attachments/assets/5ba2fc3f-fcae-4f7d-a4b9-c557d8760800" />